src/Card/Card.css

```diff
@@ -0,0 +1,41 @@
+.area-card {
+  width: 35%;
+  height: 60%;
+  text-align: center;
+  background: #FFFFFF;
+  border-radius: 15px;
+  padding: 1%;
+  margin: 2%;
+  margin-left: 7%;
+}
+
+.card-container {
+display: flex;
+flex-wrap: wrap;
+width: 100vw;
+height: 100vh;
+background: rgba(26, 30, 37, 0.93);
+}
+
+.areas-header {
+align-items: center;
+background-color: white;
+display: flex;
+justify-content: space-between;
+width: 100vw;
+padding-left: 4%;
+}
```

Comment:
This looks good! Let's make sure to clean up indentation and alphabetize all properties before converting to Sass/Scss or turning in. It will make it easier to see patterns and repeated code so we know what can be combined.